mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_Mobile.java

```diff
@@ -163,6 +170,16 @@ protected void onCreate(final Bundle savedInstanceState) {
             //Handle the IllegalArgumentException
         }
 
+        // Define webview user agent for known external hosts
+        final String defaultUA = WebSettings.getDefaultUserAgent(this);
+        final String mIITCDefaultUA = defaultUA.replace("; wv", "");
+        final String googleUA = (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) ? mDesktopUA : mIITCDefaultUA;
+
+        mAllowedHostnames.put("intel.ingress.com", mIITCDefaultUA);
+        mAllowedHostnames.put("google.com", googleUA);
+        mAllowedHostnames.put("youtube.com", googleUA);
```

Comment:
youtube? Why?

Reply:
it was in the whitelist for google login.I don't know why.